mapper/pchain/tx_parser.go

```diff
@@ -109,18 +107,28 @@ func (t *TxParser) Parse(txID ids.ID, tx txs.UnsignedTx) (*types.Transaction, er
 		return nil, err
 	}
 
+	txMetadata := map[string]interface{}{
+		MetadataTxType: txType,
+	}
+
+	if ops.ImportIns != nil {
```

Comment:
You should add a check if `ops` is not nil for no-op transactions such as AdvanceTime